go/pkg/client/cas.go

```diff
@@ -113,6 +113,9 @@ func (c *Client) UploadIfMissing(ctx context.Context, data ...*chunker.Chunker)
 
 	missing, present, err := c.getMissingPresent(ctx, newUploads)
 	if err != nil {
+		for _, d := range newUploads {
```

Comment:
nit: have a locally defined cleanup function and call it here and down below?

Reply:
Oh, right, I have that function! Why didn't I call it?...